src/index.html

```diff
@@ -1,13 +1,332 @@
+            <ul class="nav__list">
+              <li class="nav__item nav__item--active">
+                <a href="#" class="nav__link">Home</a>
+              </li>
+              <li class="nav__item">
+                <a href="#info" class="nav__link">About us</a>
+              </li>
+              <li class="nav__item">
+                <a href="#gallery" class="nav__link">Gallery</a>
+              </li>
+              <li class="nav__item">
+                <a href="#callback" class="nav__link">Consultation</a>
+              </li>
+              <li class="nav__item">
+                <a href="#clients" class="nav__link">Testimonials</a>
+              </li>
+              <li class="nav__item">
+                <a href="#contacts" class="nav__link">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+
+      </div>
+    </header>
+
+    <main class="container">
+
+      <section class="section info" id="info">
+        <h2 class="info__title">Miami best<br>Real estate <a href="https://goo.gl/maps/anCFi2RZg2mPREBo7">here</a></h2>
+        <article class="info__article">
+          <h3>Over 50 years experience in love.</h3>
+          <p>Mazel and VA Tsukkerman in his "Analysis of musical works."<br> Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+          <p>The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information about the profitable deals.</p>
+          <p><strong>We will never be late!<br>Guaranteed secure transaction - 100%</strong></p>
+        </article>
+      </section>
+
+      <section class="section gallery" id="gallery">
+        <h2 class="gallery__title">Gallery</h2>
+        <img
+          class="small-logo"
+          src="images/crown.svg"
+          alt="Small logo"
+        >
+        <div class="gallery__pictures">
+          <div class="gallery__row gallery__row--wide-left">
+            <img
+              src="images/photos/1.jpg"
+              alt="Marina Palms residence"
+              class="gallery__picture"
+            >
+            <img
+              src="images/photos/2.jpg"
+              alt="Marina Palms residence"
+              class="gallery__picture"
+            >
+          </div>
+          <div class="gallery__address">
+            <a href="https://goo.gl/maps/anCFi2RZg2mPREBo7">
+              <strong>Marina Palms</strong>
+            </a>
+            <span> /North Miami Beach, FL 33162</span>
+          </div>
+
+          <div class="gallery__row gallery__row--wide-even">
+            <img
+            src="images/photos/3.jpg"
+            alt="Edition Residens"
+            class="gallery__picture"
+            >
```

Comment:
атрибуты должны иметь два пробела отступа при переносе
```suggestion
            <img
              src="images/photos/3.jpg"
              alt="Edition Residens"
              class="gallery__picture"
            >
```

исправь везде